index.html

```diff
@@ -163,7 +165,7 @@
 </script>
 </head>
 <body>
-<div id="plaintext-wrapper"><textarea id="plaintext" spellcheck="false"></textarea></div>
+<div id="plaintext-wrapper"><textarea id="plaintext" spellcheck="false" placeholder="Please enable JavaScript."></textarea></div>
```

Comment:
I support the idea, but the canonical way is to use `noscript` element.

Reply:
To change `placeholder`? I would not know how to without using js.